content/tutorials/newsfeed-react.textile

```diff
@@ -0,0 +1,450 @@
+---
+title: React Live NewsFeed Tutorial
+section: tutorials
+index: 90
+---
+
+In this tutorial, we will see how to use Ably "Realtime client library":/realtime to build a realtime NewsFeed with React. We'll be making use Ably's public channel. Once a update is posted, we'll publish it to the public channel. We will also be subscribed to the channel so we can see updates as they are added in realtime.
+
+This tutorial will not include any server-side implementation. Hence, updates won't be persisted to any storage.
+
+React is a JavaScript library for building user interfaces. It is component-based and declarative which makes it easy to build complex UI with it. For more information on React, please refer to their official website "https://reactjs.org":https://reactjs.org.
+
+<img src="/images/tutorials/react-commenting/react_logo.png" style="width: 50%" alt="React logo">
+
+<%= partial 'tutorials/_step-1-setup-free-account' %>
+
+h2. Step 2 - Create a React App
+
+We'll start by creating a new React app. To do this, we'll be using "create-react-app":https://github.com/facebook/create-react-app. create-react-app allows you to create a React app without having to worry about build configurations.
```

Comment:
In textile, all commands and method definitions etc are formatted with `@@`, please enclose the `create-react-app` command within that. Also, it's worth mentioning that it's a command, which otherwise makes it unclear.